### PR TITLE
Added BZ#1791916 as a known issue

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1791916

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@mffiedler PTAL

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -916,3 +916,26 @@ fail.
 * The public cloud access image for RHEL cannot be used directly if you enable FIPS mode. This is caused by public cloud images not allowing kernel integrity checks. To do this, you must upload your own images. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1788051[*BZ#1788051*])
 * The Operator Lifecycle Manager (OLM) does not work in {product-title} when Kuryr SDN is enabled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1786217[*BZ#1786217*])
 * The `oc adm catalog build` and `oc adm catalog mirror` commands do not work for the restricted cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1773821)[*BZ#1773821*])
+
+* When upgrading a {product-title} cluster from  4.1 to 4.2 to 4.3, there is a
+possibility that the Node Tuning Operator tuned pods can get stuck in
```
Comment (codyhoag) [inline release_notes/ocp-4-3-release-notes.adoc]: Suggestion: adding `the` in front of `ContainerCreating state` may offer a more natural flow.

File: release_notes/ocp-4-3-release-notes.adoc [hunk 2]
```diff
@@ -916,3 +916,26 @@ fail.
...
++
+To resolve the issue, apply the following workaround. Run:
++
+----
+$ oc delete daemonset/tuned -n openshift-cluster-node-tuning-operator
+$ oc get daemonset/tuned -n openshift-cluster-node-tuning-operator
+$ oc get pods -n openshift-cluster-node-tuning-operator
+----
++
+Verify that the Pods are now in a `Running` state.
```
Comment (codyhoag) [inline release_notes/ocp-4-3-release-notes.adoc]: Should `Pods` be capitalized? If so, the two lower-cased `pods` used above will need to be updated.

**Review by codyhoag — COMMENTED** [review]
Just a couple minor wordsmithing suggestions. Thanks for adding this in!

File: release_notes/ocp-4-3-release-notes.adoc (hunk 2 above)
Comment (mffiedler) [reply release_notes/ocp-4-3-release-notes.adoc]: I think lower case pods is usual.   Example:  https://docs.openshift.com/container-platform/4.2/nodes/pods/nodes-pods-viewing.html

Comment (ahardin-rh) [reply release_notes/ocp-4-3-release-notes.adoc]: @codyhoag @mffiedler For 4.x docs, we do capitalize Pods: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting
I'll do a doc sweep as a follow-up to this to ensure we're consistent.

File: release_notes/ocp-4-3-release-notes.adoc (hunk 1 above)
Comment (mffiedler) [reply release_notes/ocp-4-3-release-notes.adoc]: s/pods/Pods

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**